llvm/lib/Support/raw_ostream.cpp

```diff
@@ -906,11 +912,14 @@ raw_fd_ostream &llvm::outs() {
 }
 
 raw_fd_ostream &llvm::errs() {
-  // Set standard error to be unbuffered and tied to outs() by default.
+  if (auto *TLSErrs = OutsOverride; TLSErrs != nullptr)
```

Comment:
There's a copy-paste error here, should be `ErrsOverride`. Suprised that the test below hasn't catched that, maybe add another test that writes to different buffers?